scipy/stats/_continuous_distns.py

```diff
@@ -5206,8 +5206,133 @@ def _entropy(self, beta):
 halfgennorm = halfgennorm_gen(a=0, name='halfgennorm')
 
 
+class rv_histogram(rv_continuous):
+    """
+    Generates a distribution given by a histogram.
+    This is useful to generate a template distribution from a binned datasample.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    There are no additional shape parameters except for the loc and scale.
+    The pdf is defined as a stepwise function from the provided histogram
+    The cdf is a linear interpolation of the pdf.
+
+    %(after_notes)s
+
+    Examples
+    --------
+
+    Create a scipy.stats distribution from a numpy histogram
+    >>> data = scipy.stats.norm.rvs(size=100000, loc=0, scale=1.5, random_state=123)
+    >>> hist = np.histogram(data, bins=100)
+    >>> hist_dist = scipy.stats.rv_histogram(hist)
+
+    Behaves like an ordinary scipy rv_continuous distribution
+    >>> hist_dist.pdf(1.0)
+    3.5
```

Comment:
This number is a placeholder,
I didn't find out yet, howto run the docstring tests. :-)